common/src/main/java/dev/ftb/mods/ftbultimine/FTBUltiminePlayerData.java

```diff
@@ -40,7 +40,7 @@ public void clearCache() {
 
 	public static HitResult rayTrace(ServerPlayer player) {
 		double distance = PlatformMethods.reach(player);
-		return player.pick(player.isCreative() ? distance : distance - 0.5D, 1F, false);
+		return player.pick(player.isCreative() ? distance + 0.5D : distance, 1F, false);
```

Comment:
So there is a difference between survival and creative reach distance, but they seem to have been reversed.